client/me/purchases/remove-purchase/index.jsx

```diff
@@ -207,6 +213,53 @@ class RemovePurchase extends Component {
 		);
 	}
 
+	renderDomainMappingDialog() {
+		const { purchase, site } = this.props;
+
+		return (
+			<CancelPurchaseForm
+				disableButtons={ this.state.isRemoving }
+				defaultContent={ this.renderDomainMappingDialogText() }
+				purchase={ purchase }
+				selectedSite={ site }
+				isVisible={ this.state.isDialogVisible }
+				onClose={ this.closeDialog }
+				onClickFinalConfirm={ this.removePurchase }
+				flowType={ CANCEL_FLOW_TYPE.REMOVE }
+			/>
+		);
+	}
+
+	renderDomainMappingDialogText() {
+		const { purchase, translate } = this.props;
+		const productName = getName( purchase );
+		const domainProductName = purchase.productName;
+
+		return (
+			<div>
+				<p>
+					{
+						/* translators: "productName" is a product name (in this case, Domain Mapping). "domain" is something like example.wordpress.com */
```

Comment:
I think this translator comment is incorrect. `productName` in this message is the domain name.

This isn't the fault of the PR, but rather the complex way we handle domain product naming. I think the variables could be cleared up here. Can you do the following: 

- update the variable on L235 to be `domainName` since this render is very specific (and `productName` followed by a different variable that is set with `purchase.productName` is somewhat confusing) 
- update the translation component to be `site` (instead of `domain`)
- update the translators comment to reflect the variable naming
- remove the `if we want to use the purchased domain…` comment

Then, in the next translation string (L251):

- make the translator comment format consistent between the two strings (either as a `comment` parameter or a `/* translators:` comment block - I think the latter is preferred now)
- remove the unused `productName` arg from this.

Also, it's not really important, but I believe that the "Domain Mapping" product name was updated to "Domain Connection" this week by the Domains team, so while you're in there, you might want to change the comment strings (I'd leave the function naming alone, since our codebase still refers to the product almost exclusively as domain mapping - it's more a display name change).